data/json/effects.json

```diff
@@ -676,6 +676,60 @@
             "sleep_chance": [101]
         }
     },
+    {
+        "type": "effect_type",
+        "id": "bandaged",
+        "name": ["Bandaged"],
+        "desc": ["Your wounds are bandaged."],
+        "main_parts_only": true,
+        "rating": "good",
+        "int_dur_factor": 3600,
+        "max_intensity": 16,
+        "//": "How many intensity levels may apply effects",
+        "max_effective_intensity": 8,
+        "max_duration": 57600,
+        "base_mods": {
+            "//": "Default healing rate (x100000)", 
```

Comment:
This shouldn't be in the json itself but in docs. Otherwise modders are likely to pollute the jsons with copypasted comments.